waveorder/acq/acq_functions.py

```diff
@@ -164,8 +164,8 @@ def acquire_from_settings(
         files = glob.glob(path + "*")
         index = max([int(x.split(path + "_")[1]) for x in files])
 
-        reader = read_micromanager(
+        reader = read_images(
             path + f"_{index}", data_type="ometiff", extract_data=True
         )
 
-        return reader.get_array(0)
+        return reader[0].compute()
```

Comment:
Is this correct? I know you're unable to test pycromanager, but will this return an array?

Reply:
Now using the MMStack reader directly so the LSP can infer the return type here.